bootstrap-extensions/src/main/java/de/agilecoders/wicket/extensions/markup/html/bootstrap/form/datetime/DatetimePickerWithIcon.html

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<wicket:panel xmlns:wicket="http://wicket.apache.org">
+    <div wicket:id="dateWrapper" class="input-group date">
+        <input type="text" class="form-control" wicket:id="date"/>
+        <span class="input-group-addon"><i class="glyphicon glyphicon-calendar"></i></span>
```

Comment:
May we make the icon configurable ? I.e. with a Wicket component.
I (and many others) use FontAwesome in my app and I'd prefer to use `fa-calendar` instead.